draft-kwbdgrr-tsvwg-net-collab-rqmts.md

```diff
@@ -286,44 +263,30 @@ In this document, this is termed 'Reactive Management'.
-manages QoS of flows of multiple users and can buffer (delay),
-discard, or apply other flow control rules. Application layer
-signaling and feedback between Client - Server (A - in the figure)
-adjust transmission rate over a period of several RTTs using feedback
-and congestion control algorithms.  Congestion control algorithms (CCAs)
-are generally conservative and settle to a steady rate that avoids
-excessive packet loss.  In networks where link conditions (between
-Client and Router) vary significantly at timescales well below the
-RTT, this results in unused (wasted) bandwidth at short timescales.
-
-There is some research (e.g., {{5G-Octopus}}) to indicate that media
-applications can obtain better Quality of Experience (QoE) when sending at a higher rate
-(less conservative than current CCA) and the media application is
-willing to tolerate some packet loss or delay of low priority
-packets.  Packet priority and tolerance to delay of packets in such
-a case would be provided on-path in a side channel associated to
-the downstream packet (B - in the figure).  The requirements for this
-server-to-network (S2N) metadata are described in {{server-network}}.
-
-The client may provide information to an (access) router to drop
-lower priority marked packets of a flow (UDP 4-tuple) temporarily
-which can in turn allocate available bandwidth to other flows of that network
-attachment, especially during a reactive management event.
-
-Network shapers
-observe flows and apply policies to maximize performance but are
-not aware whether  there is a high preference for one flow (UDP
-4-tuple) over another flow belonging to the same user and network
-attachment (e.g., a subscriber connection, a 3GPP PDU Session. See
-{{net-attach}} for more details).  Clients may provide an (access)
-router with preferences of which flow (UDP 4-tuple) has relative
-priority among flows of that network attachment using H2N (C - in
-the figure).  Clients may also provide information to an (access) router
-to drop, when resources are congested,  'lower priority'-marked
-packets of a flow (UDP 4-tuple) temporarily which can in turn
-allocate bandwidth to other flows of that same network attachment.
+{{Figure-netshaper}} shows a bottleneck (access) router on the Server to Client path.
+A network shaper in the router manages QoS of multiple users’ flows and can buffer,
+discard, or apply other flow control rules. Application layer signaling and feedback
+between Client and Server (A - in the figure) adjust transmission rate over several RTTs
+using feedback and congestion control algorithms (CCAs). CCAs are generally conservative
+and settle to a steady rate to avoid excessive packet loss. In networks where link
+conditions (between Client and Router) vary significantly at sub-RTT timescales,
+this results in unused bandwidth at short timescales.
+
+Research (e.g., {{5G-Octopus}}) indicates that media applications can achieve better QoE
```

Comment:
```suggestion
Research (e.g., {{5G-Octopus}}) indicates that media applications can achieve better Quality of Experience (QoE)
```

IETF likes acronym expanded on first use. We don't use QoE later, but it's useful to have **QoE** in the document if someone searches for it.